src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/20200930085741_EES-1385_Update_FilteredObservations_SP.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
+using static GovUk.Education.ExploreEducationStatistics.Data.Model.Migrations.MigrationConstants;
+
+namespace GovUk.Education.ExploreEducationStatistics.Data.Model.Migrations
+{
+    public partial class EES1385_Update_FilteredObservations_SP : Migration
+    {
+        private const string MigrationId = "20200930085741";
+        private const string PreviousInsertObservationFilterItemsMigrationId = "20200304145743";
+        private const string PreviousFilteredObservationsMigrationId = "20200308154804";
+        
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.Sql("DROP PROCEDURE dbo.InsertObservationFilterItems");
+            migrationBuilder.Sql("DROP TYPE dbo.ObservationFilterItemType");
+            migrationBuilder.Sql("DROP PROCEDURE dbo.FilteredObservations");
+
+            migrationBuilder.SqlFromFile(MigrationsPath, $"{MigrationId}_Type_ObservationFilterItemType.sql");
+            migrationBuilder.SqlFromFile(MigrationsPath, $"{PreviousInsertObservationFilterItemsMigrationId}_Routine_InsertObservationFilterItems.sql");
+            migrationBuilder.SqlFromFile(MigrationsPath, $"{MigrationId}_Type_FilterTableType.sql");
+            migrationBuilder.SqlFromFile(MigrationsPath, $"{MigrationId}_Routine_GetFilteredObservations.sql");
+            migrationBuilder.SqlFromFile(MigrationsPath, $"{MigrationId}_Routine_GetObservations.sql");
+        }
+
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.Sql("CREATE TYPE ObservationFilterItemType AS table( ObservationId uniqueidentifier NOT NULL, FilterItemId  uniqueidentifier NOT NULL)");
+            migrationBuilder.SqlFromFile(MigrationsPath, $"{PreviousInsertObservationFilterItemsMigrationId}_Routine_InsertObservationFilterItems.sql");
+            migrationBuilder.Sql("DROP PROCEDURE dbo.GetFilteredObservations");
+            migrationBuilder.Sql("DROP PROCEDURE dbo.GetObservations");
+            migrationBuilder.Sql("DROP TYPE dbo.FilterTableType");
+            migrationBuilder.SqlFromFile(MigrationsPath, $"{PreviousFilteredObservationsMigrationId}_Routine_FilteredObservations.sql");
```

Comment:
I don't think this exists

Reply:
I assume you mean 20200308154804_Routine_FilteredObservations.sql ? Yes it exists

Reply:
Sorry, I got the wrong Id. 😄